clinica/pipelines/t1_freesurfer_longitudinal/t1_freesurfer_longitudinal_cli.py

```diff
@@ -0,0 +1,155 @@
+# coding: utf8
+
+import clinica.engine as ce
+
+__author__ = "Alexis Guyot"
+__copyright__ = "Copyright 2016-2019, The Aramis Lab Team"
+__credits__ = ["Alexis Guyot"]
+__license__ = "See LICENSE.txt file"
+__version__ = "0.1.0"
+__maintainer__ = "Alexis Guyot"
+__email__ = "alexis.guyot@icm-institute.org"
+__status__ = "Development"
+
+
+
+def build_pipeline(pipeline):
```

Comment:
I'm now using the [pipeline].build() commands as per your comment.

I can't quite remember why I created my own build_pipeline()/run_pipeline() commands. It think it has to do with the fact I'm having an overall workflow that connects my two pipelines, and I use the run() method on the workflow instead of applying a run() successively to t1_freesurfer_template and to t1_freesurfer_longitudinal_correction. This means the method check_size() will not be called at any time in my case. I suspect I wanted to move the check_size() method to the end of my custom-created build_pipeline() method.